vlog4j-examples/src/main/java/org/semanticweb/vlog4j/examples/core/ConfigureReasonerLogging.java

```diff
@@ -69,11 +69,11 @@ public static void main(final String[] args) throws IOException, ParsingExceptio
 		try (final Reasoner reasoner = Reasoner.getInstance()) {
 			final KnowledgeBase kb = reasoner.getKnowledgeBase();
 			/* exists z. B(?y, !z) :- A(?x, ?y) . */
-			kb.addStatements(RuleParser.parseRule("B(?Y, !Z) :- A(?X, ?Y) ."));
+			RuleParser.parseInto(kb, "B(?Y, !Z) :- A(?X, ?Y) .");
 			/* B(?y, ?x), A(?y, ?x) :- B(?x, ?y) . */
-			kb.addStatements(RuleParser.parseRule("B(?Y, ?X), A(?Y, ?X) :- B(?X, ?Y) ."));
+			RuleParser.parseInto(kb, "B(?Y, ?X), A(?Y, ?X) :- B(?X, ?Y) .");
```

Comment:
this one as well, I don't know, maybe the parse rule is not important as well and we can just rely on parseinto

Reply:
`RuleParser.parseRule` is still part of the code.